package.json

```diff
@@ -153,7 +153,7 @@
     "css-loader": "5.2.6",
     "dotenv-webpack": "7.0.3",
     "enzyme": "3.11.0",
-    "enzyme-adapter-react-16": "1.15.6",
+    "enzyme-adapter-react-17-updated": "1.0.2",
```

Comment:
I didn't realize Enzyme didn't support React 17 (https://github.com/enzymejs/enzyme/issues/2429) or 18 (https://github.com/enzymejs/enzyme/issues/2524) itself, and that it hasn't been updated in quite a while (the last release of Enzyme was in December 2019). I'm going to file a ticket to start investigating switching away from Enzyme since it seems like it's no longer being properly supported

Reply:
https://mattermost.atlassian.net/browse/MM-45873